jwala-services/src/main/java/com/cerner/jwala/service/configuration/service/AemServiceConfiguration.java

```diff
@@ -191,21 +191,23 @@ public BalancerManagerService getBalancermanagerService(final GroupService group
                                                             final ClientFactoryHelper clientFactoryHelper,
                                                             final HistoryFacadeService historyFacadeService) {
         return new BalancerManagerServiceImpl(groupService, applicationService, webServerService, jvmService, clientFactoryHelper,
-                                              new BalancerManagerHtmlParser(), new BalancerManagerXmlParser(jvmService),
-                                              new BalancerManagerHttpClient(), historyFacadeService);
+                new BalancerManagerHtmlParser(), new BalancerManagerXmlParser(jvmService),
+                new BalancerManagerHttpClient(), historyFacadeService);
     }
 
     @Bean(name = "webServerService")
     public WebServerService getWebServerService(final ResourceService resourceService,
                                                 @Qualifier("webServerInMemoryStateManagerService")
                                                 final InMemoryStateManagerService<Identifier<WebServer>, WebServerReachableState> inMemoryStateManagerService,
-                                                @Value("${paths.resource-templates:../data/templates}") final String templatePath) {
+                                                @Value("${paths.resource-templates:../data/templates}") final String templatePath,
+                                                final JvmPersistenceService jvmPersistenceService) {
```

Comment:
Although I prefer constructor injection, I remember the team decided to do class variable auto-wiring instead to keep the constructor from having too many parameters (which is a good point) 

    class WebServerServiceImpl 
       
        @Autowired
        private JvmPersistenceService jvmPersistenceService;

        ....

    }

However, please confirm with the others if you can do constructor injection for now since we're in a tight deadline